sysvar: add a note to disable tidb_enable_rate_limit_action (#11444)
pingcap/docs: configure-memory-usage.md

- TiDB supports dynamic memory control for the operator that reads data. By default, this operator uses the maximum number of threads that [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) allows to read data. When the memory usage of a single SQL execution exceeds [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) each time, the operator that reads data stops one thread.

> - This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default. In this situation, the memory usage might not be under the control of [`tidb_mem_quota_query`]. Therefore, it is recommended to modify the variable value to `OFF`.

Author: ```suggestion
- This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`] in some cases. Therefore, it is recommended to modify the variable value to `OFF`.
```

> - This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](/system-variable.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to modify the variable value to `OFF`.

Author: ```suggestion
- This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](/system-variable.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`.
```

> - This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](/system-variable.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`.

Author: ```suggestion
- This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`.
```

> - This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which makes the memory usage not under the control of [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to disable `tidb_enable_rate_limit_action`.

Reviewer: ```suggestion
- This flow control behavior is controlled by the system variable [`tidb_enable_rate_limit_action`](/system-variables.md#tidb_enable_rate_limit_action). This variable is enabled by default, which might make the memory usage not under the control of [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query) in some cases. Therefore, it is recommended to set the value of `tidb_enable_rate_limit_action` to `OFF`.
```
